js/apps/system/_admin/aardvark/APP/frontend/js/views/graphManagementView.js

```diff
@@ -1573,6 +1573,9 @@
     resetDisplaySettings: function () {
       var graphName = $('#editGraphName').val();
 
+      const localStorageKey = graphName + "-gv-urlparameters";
+      localStorage.removeItem(localStorageKey);
```

Comment:
change this to `window.localStorage`

Reply:
Resolved within https://github.com/arangodb/arangodb/pull/18199/commits/60e53273452d4e245f570f5c3c957f9f99d78ab7